pkg/utils/dataset/lifecycle/node.go

```diff
@@ -59,7 +59,7 @@ func AlreadyAssigned(runtimeInfo base.RuntimeInfoInterface, node v1.Node) (assig
 	exclusiveness := runtimeInfo.IsExclusive()
 	if exclusiveness {
 		log.Info("Placement Mode", "IsExclusive", exclusiveness)
-		for _, nodeLabel := range node.Labels {
+		for nodeLabel := range node.Labels {
```

Comment:
May I know the reason to change this piece of code?

Reply:
```
for _, nodeLabel := range node.Labels {
	if strings.Contains(nodeLabel, common.LabelAnnotationPrefix) {
		assigned = true
	}
}
```
In this way, it may use the value of the label to judge in the if clause rather than the key.